carma_wm/test/SignalizedIntersectionManagerTest.cpp

```diff
@@ -69,7 +69,12 @@ TEST(SignalizedIntersectionManger, convertLaneToLaneletId)
   intersection.id.id = 9001;
 
   cav_msgs::GenericLane lane;
-  lane.lane_id = 1210;
+  lane.lane_id = 1210;        //TODO: broken! implicit conversion from int to unsigned char makes this value 186. Is lane_id the wrong type???
```

Comment:
Since this is a test case, this isn't actually broken since the test scenario still seems self consistent. 
Not sure what the best comment would be here.

Reply:
I guess you have a point.  However, there are comment "diagrams" throughout this code using these IDs as well, and the overall effect is to communicate to other developers that these are legitimate lanelet IDs.  When operational code chokes on such values, people could spend a long time wondering why, when the unit tests pass.  My bigger concern is that if lanelet IDs really do take on values like this then our GenericLane message type (and maybe others?) is inadequate to represent them.  I've changed all the values to be in the 170-200 range so they are legal.  If you decide we do need to accommodate values > 256, then it would be wise to put large values back into at least one of these tests.